llvm/test/CodeGen/RISCV/GlobalISel/irtranslator/vec-alu.ll

```diff
@@ -0,0 +1,53 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
+; RUN: llc -mtriple=riscv32 -mattr=+v -global-isel -stop-before=legalizer -simplify-mir < %s | FileCheck %s --check-prefixes=CHECK,RV32I
```

Comment:
RV32I and RV64I are unused in this test. This causes FileCheck to report an error in lit. I fixed it, but you should always run `ninja check` even after running the script. The script isn't perfect.

Reply:
Thanks for fixing this. The failure didn't show up in the pre commit CI checks.